components/eamxx/scripts/test-all-scream

```diff
@@ -27,26 +48,26 @@ def parse_command_line(args, description):
 OR
 {0} --help
 
-\033[1mEXAMPLES (assumes user is on machine melvin):\033[0m
-    \033[1;32m# Run all tests on current machine using the SCREAM-approved env for this machine (this is the default env behavior) and your origin/master common ancestor for baseline generation (the default baseline behavior) \033[0m
+\033[1mEXAMPLES (assumes user is on machine mappy):\033[0m
+    \033[1;32m# Run all tests on current machine using the SCREAM-approved env for this machine \033[0m
     > cd $scream_repo/components/eamxx
-    > ./scripts/{0} -m melvin
+    > ./scripts/{0} -m mappy
 
     \033[1;32m# Run all tests on current machine with defaut behavior except using your current shell env \033[0m
     > cd $scream_repo/components/eamxx
-    > ./scripts/{0} --preserve-env -m melvin
+    > ./scripts/{0} --preserve-env -m mappy
 
-    \033[1;32m# Run all tests on current machine with default behavior except using pre-existing baselines (skips baseline generation) \033[0m
+    \033[1;32m# Run all tests on current machine with default behavior except using non-default baselines \033[0m
     > cd $scream_repo/components/eamxx
-    > ./scripts/{0} -m melvin --baseline-dir=PATH_TO_BASELINES
+    > ./scripts/{0} -m mappy --baseline-dir=PATH_TO_BASELINES
 
-    \033[1;32m# Run only the dbg test on current machine with default behavior otherwise\033[0m
+    \033[1;32m# Run all tests on current machine with default behavior except using local baselines \033[0m
     > cd $scream_repo/components/eamxx
-    > ./scripts/{0} -m melvin -t dbg
+    > ./scripts/{0} -m mappy --baseline-dir=LOCAL
```

Comment:
Should we support LOCAL at all? I'm thiking that, now that the user must explicitly specify `-g` to generate baselines, we should also ask for a folder where to put them, no? It would be one less thing to keep track/support/test in TAS. We'd only keep AUTO as special value for `-b`...

Reply:
I think LOCAL is kind of useful. If you want a temporary stash of baselines, it saves a little bit of typing.